api/envoy/extensions/filters/http/ip_tagging/v3/ip_tagging.proto

```diff
@@ -59,4 +60,7 @@ message IPTagging {
   // Tracked by issue https://github.com/envoyproxy/envoy/issues/2695]
   // The set of IP tags for the filter.
   repeated IPTag ip_tags = 4 [(validate.rules).repeated = {min_items: 1}];
+
+  // Custom header to emit tag.
```

Comment:
I'm fine with both but this deserves a more informative comment for sure. Need to answer at least (a) are we still set `x-envoy-ip-tags` if the field is non-empty (b) the header value is overridden instead of appended

Reply:
Hi @wbpcode do you have any suggestion or preference for the field name? how is `custom_header`?

Reply:
I think it's ok. (I am not good at naming anyway, haha)